cuda/test/matrix/csr_kernels.cpp

```diff
@@ -41,10 +41,15 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include <gtest/gtest.h>
 
 
+#include "core/matrix/csr_kernels.hpp"
+#include "core/matrix/dense_kernels.hpp"
+
+
```

Comment:
Private Ginkgo header (`core/test/utils.hpp` should be one of them, and also written with `""` instead of `<>`) should be below the public Ginkgo header.